scripts/drop_tables.pgsql

```diff
@@ -0,0 +1,8 @@
+drop table bdol_tx;
```

Comment:
Where is this script used?  Do we need it?

Reply:
New issue: https://github.com/bb-mvp/kafka-pipeline/issues/40#issue-1537926115.
Will be subject to another PR